Yoast/javascript
# Prepare Swedish morphology for 15.2 release

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds and refactors an external stemmer for Swedish
* [Yoast SEO Premium] Adds word forms support for Swedish keyphrase recognition.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In your testing environment, set the language to Swedish
* Make sure to test this functionality in premium
* Add text minimum 150 words
* Use bedrag as the focus keyphrase
* Add these other forms of the word in the text field bedraga, bedragare, bedragen, bedrager and make sure that all these forms are recognized.

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-585
